index.html

```diff
@@ -0,0 +1,52 @@
+<link rel="stylesheet" href="css/style.css">
+<link href="https://fonts.googleapis.com/css?family=Poppins:200i,300,400&amp;display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <nav id="navbar">
+    
+       <a href="#welcome-section">About</a>
+       <a href="#projects">Work</a>
+       <a href="#conact">Contact</a>
+    
+
+  </nav>
```

Comment:
good job noticing this error 👍 however I think it would be better to phrase the comment more politely in order to keep the communication with the student more friendly, please refer to my previous comment for an example.